metaci/build/migrations/0019_build_planrepo.py

```diff
@@ -10,7 +10,7 @@ def set_planrepository(apps, schema_editor):
     Build = apps.get_model('build', 'Build')
     for build in Build.objects.all().iterator():
         try:
-            build.planrepo = build.plan.planrepository_set.get(repo = build.repo)
+            build.planrepo = build.plan.planrepos.get(repo = build.repo)
```

Comment:
do search and replace to make sure we caught all uses of planrepository_set

Reply:
This one's actually wrong, per our discussion earlier, @bethbrains - this is the other side of the "junction object" relationship. I'll push a fix.

Reply:
I'd advocate for changing the name on that side too so it's consistent, but understand if you don't want to add to the scope of this PR.

Reply:
Done.